types/arc4/arc4-tests.ts

```diff
@@ -0,0 +1,24 @@
+import arc4, { lodash, normal } from "arc4";
```

Comment:
you could try:
```ts
import arc4 = require("arc4");
const { lodash, normal } = arc4;
```
(that's only needed on DT, package can be consumed in common way with default TS configs)